lib/blockchain.go

```diff
@@ -2579,7 +2581,30 @@ func (bc *Blockchain) CreatePrivateMessageTxn(
 
 		// Add {V : 2} version field to ExtraData to indicate we are
 		// encrypting using shared secret.
-		messageExtraData["V"] = UintToBuf(2)
+		messageExtraData[MessagesVersionString] = UintToBuf(MessagesVersion2)
```

Comment:
I don't think we can really refactor the `V` ExtraData field here. All V2 messages would break from this, because they are already stored with the `V` field.